src/Mvc/Mvc.Testing/src/Handlers/RedirectHandler.cs

```diff
@@ -152,9 +152,10 @@ private static async Task<(Stream originalBody, Stream copy)> CopyBody(HttpReque
                 redirect.Content = originalContent;
             }
 
-            foreach (var property in response.RequestMessage.Properties)
+            foreach (var property in response.RequestMessage.Options)
```

Comment:
/cc @Tratcher, they obsoleted `Properties` so I moved to the suggested "Options".